src/utils/callbackSet.ts

```diff
@@ -0,0 +1,45 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+  export module Utils {
+    /**
+     * A set of callbacks which can be all invoked at once.
+     * Each callback exists at most once in the set (based on reference equality).
+     * Ideally, all callbacks should have the same signature.
```

Comment:
Isn't it not even an "ideally"?

I think the reference equality is worth noting, but I don't think we need to explain that each callback exists at most once -- that's the definition of a "set", after all.